packages/trader/src/Modules/Reports/Components/filter-component.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import CompositeCalendar from 'App/Components/Form/CompositeCalendar/composite-calendar.jsx';
+
+const TransactionFilter = ({ handleFilterChange }) => {
+    const [default_filter, setDefaultFilter] = React.useState('all');
```

Comment:
We should initialize the filter with selected `action_type` from store. When visiting the profit table/statement again after navigating to contract page, it again filters with the last selected date and `action_type`